Src/HoneyBear.HalClient/project.json

```diff
@@ -0,0 +1,36 @@
+﻿{
+  "version": "1.0.0-*",
+  "title": "HoneyBear.HalClient",
+  "description": "A lightweight fluent .NET client for navigating and consuming HAL APIs.",
+  "language": "en-US",
+  "authors": [ "Eoin O'Connor" ],
+  "copyright": "Copyright © Eoin O'Connor 2016",
+  "packOptions": {
+    "owners": [ "Eoin O'Connor" ],
+    "projectUrl": "https://github.com/eoin55/HoneyBear.HalClient",
+    "tags": [ "HAL", "JSON", "Hypermedia", "HATEOAS", "REST" ],
+    "licenseUrl": "https://github.com/eoin55/HoneyBear.HalClient/LICENSE",
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/eoin55/HoneyBear.HalClient"
+    }
```

Comment:
Could you add the following, please?

```
  "buildOptions": {
    "xmlDoc": true
  }
```

This should generate the XML docs, which makes the library easier to consume.